src/download.rs

```diff
@@ -193,12 +200,13 @@ fn unpack(tar_path: &Path, dst: &Path) -> Result<()> {
     Ok(())
 }
 
-pub fn download_file(url: &str, path: &PathBuf) -> Result<()> {
+pub fn download_file(url: &str, path: &PathBuf, hasher: Option<&mut Sha256>) -> Result<()> {
```

Comment:
It looks like there are no cases where we download a file without hashing it. Maybe we can change this to:

```suggestion
pub fn download_file(url: &str, path: &PathBuf, hasher: &mut Sha256) -> Result<()> {
```

Reply:
Seems like so for now, so I'll remove it. Initially put it there as an `Option` so that we have the option (pun intended) to use it or not